modules/apps/foundation/portal-configuration/portal-configuration-cluster/src/main/java/com/liferay/portal/configuration/cluster/internal/messaging/ConfigurationMessageListener.java

```diff
@@ -96,7 +91,7 @@ protected void reloadConfiguration(String pid, String filter, int type)
 					configuration.delete();
 				}
 				else {
-					configuration.update(properties);
+					configuration.update();
```

Comment:
@drewbrokke I tested this again:

1. With ```configuration.update(properties)```, the configuration entry in the database gets nulled out.

2. With ```configuration.update()```, the configuration in the database is updated correctly and both nodes have the correct configuration displaying in the UI.

I believe when passing the properties, it's actually using those to update the database causing the issue. Since the database has already been updated, the nodes receiving the cluster message simply need to reload the configuration and not try to update it again.

If you'd like to see it in action, just let me know.